File: GuessMyMusicAPI/src/main/java/guessMyMusicApp/beans/Genre.java
package guessMyMusicApp.beans;

import java.util.List;

public class Genre {

    private String name;
    private String infoText;
    private List<String> interpretes;

    public Genre (String name, String infoText, List<String> interpretes) {
        this.name = name;
        this.infoText = infoText;
        this.interpretes = interpretes;
    }

    public Genre() {

    }

    public String getName() {
        return name;
    }

    public String getInfoText() {
        return infoText;
    }

    public List<String> getInterpretes() {
        return interpretes;
    }

}

File: Apps/GuessMyMusic/GuessMyMusic/Models/Genre.cs
using System;
namespace GuessMyMusic.Models
{
    public class Genre
    {
        private string name;
        private string bpmRange;

        public string BpmRange { get => bpmRange; set => bpmRange = value; }
        public string Name { get => name; set => name = value; }

        public Genre(string name, string bpmRange)
        {
            this.Name = name;
            this.BpmRange = bpmRange;
        }
    }
}
